assets/javascripts/discourse/templates/components/wizard-custom-action.hbs

```diff
@@ -283,6 +283,62 @@
   </div>
 {{/if}}
 
+{{#if watchCategories}}
+  <div class="setting full field-mapper-setting">
+    <div class="setting-label">
+      <label>{{i18n "admin.wizard.action.watch_categories.categories"}}</label>
+    </div>
+    
+    <div class="setting-value">
+      {{wizard-mapper
+        inputs=action.categories
+        property='categories'
+        onUpdate=(action 'mappedFieldUpdated')
+        options=(hash
+          textSelection='key,value'
+          wizardFieldSelection=true
```

Comment:
We need to at least have "category" selection in the "output" here, so that an admin can set, or conditionally set, a list of categories. Also probably worth having userField selection in the key and value fields, to allow for category watching to be set according to user properties (this is directly relevant to the usecase of at least one forum that is waiting on this feature).

Reply:
as discussed I believe this is resolved.